charts/dex/templates/config.yaml

```diff
@@ -29,18 +29,18 @@ data:
         clientSecret: $GITHUB_CLIENT_SECRET
         redirectURI: {{ .Values.config.dexRedirectURI }}
         orgs:
-        - name: conde-nast-international
+        - name: yldio
     oauth2:
       skipApprovalScreen: true
     enablePasswordDB: false
     staticClients:
-    - id: k8s-auth
+    - id: kubernetes-auth
       redirectURIs:
 {{ toYaml .Values.config.authRedirectURIs | indent 6 }}
       name: 'Kubernetes authentication'
-      secret: ZXhhbXBsZS1hcHAtc2VjcmV0
```

Comment:
this is public, so should be rebased

Reply:
> this is public, so should be rebased

Its not a real secret, it was autogenerated.